### PR TITLE
[Live-Site] Add warning modal before event end

### LINKED ISSUE #685: [Live-Site]: Add warning modal before event end.
### Description
Currently, when the event ends it will abruptly kick out all the peers in the event, and the event ends. As a part of this story, we will add the warning modal before the event ends. So, that host will be sure that he/she wants to end the event.

### AC
- Waring modal open when clicked on end event.
- The warning message will be "Are you sure to end this event?"
- Two action buttons will be there with "Cancel" And "End"

### UX
https://www.figma.com/file/2VtPGJH61Qp0iarkKW6KTO/Live-Site-Detailed-Designs?type=design&node-id=0-1&mode=design&t=7omNkioL533Jwbzt-0

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### What is the change?
Added warning modal before the host ends the event.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

https://github.com/Real-Dev-Squad/website-www/assets/40385118/a587f6a9-2b70-455e-8fe5-15e619c5ebc3

https://github.com/Real-Dev-Squad/website-www/assets/40385118/b10c69c1-8f66-4f94-8914-4a17ba1586bf

### Responsiveness
modal

### CODE REVIEW

**Review by Pratiyushkumar — COMMENTED** [review]
can you please write the tests for it and share test coverge

**Review by Achintya-Chatterjee — COMMENTED** [review]
@SanketDhabarde please add tests stats

File: app/components/kickout-modal.hbs [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
       @onClick={{this.closeModal}}
     />
     <Reusables::Button
-      @text='Remove'
+      @text={{@confirmText}}
       @variant='pink btn-dark btn--sm'
       @test='kickout-button-remove'
       @type='button'
-      @onClick={{this.removePeer}}
+      @onClick={{this.confirmFun}}
```
Comment (satyam73) [inline app/components/kickout-modal.hbs:15]: confirmFun is not looking good in naming, what about `onConfirmClick` or just `onConfirm`

**Review by iamitprakash — DISMISSED** [review]

**Review by Pratiyushkumar — DISMISSED** [review]
LGTM

**Review by satyam73 — CHANGES_REQUESTED** [review]

**Comment by satyam73** [comment]
Can you please add mobile version of modal too?
Upto 280px it should work

File: app/components/kickout-modal.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (SanketDhabarde) [reply app/components/kickout-modal.hbs:15]: Now that I think about it. I guess `confirmHandler` will be good. what do you think?

**Comment by SanketDhabarde** [comment]
> Can you please add mobile version of modal too? Upto 280px it should work

It's the same as kickout modal. No change.

File: app/components/kickout-modal.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (satyam73) [reply app/components/kickout-modal.hbs:15]: works

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**